script.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/sh
+
+initial_workload_ns=""
+pvc_name="dd-io-pvc-nf" # nf stands for noflag for dd
+unset pvc_snapshot_array
+unset pod_array
+unset pending_pod_array
+
+echo "Switching project to $initial_workload_ns"
+oc project $initial_workload_ns
+
+
+echo "Creating Snapshot"
+for i in {1..25}
+do
+    echo "apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: $pvc_name-$i-snapshot
+spec:
+  volumeSnapshotClassName: ocs-storagecluster-rbdplugin-snapclass
+  source:
+    persistentVolumeClaimName: $pvc_name-$i
+" | kubectl apply -f -
+
+done
+
```

Comment:
Sleep for some duration before checking snapshot status (same between most stages in the script)

Reply:
done